packages/actions/src/Commands/Concerns/CanGenerateImporterColumns.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Filament\Actions\Commands\Concerns;
+
+use Doctrine\DBAL\Types;
+use Illuminate\Support\Str;
+
+trait CanGenerateImporterColumns
+{
+    protected function getImporterColumns(string $model): string
+    {
+        $model = $this->getModel($model);
+
+        if (blank($model)) {
+            return '//';
+        }
+
+        $table = $this->getModelTable($model);
+
+        if (blank($table)) {
+            return '//';
+        }
+
+        $columns = [];
+
+        foreach ($table->getColumns() as $column) {
+            if ($column->getAutoincrement()) {
+                continue;
+            }
+
+            $columnName = $column->getName();
+
+            if (str($columnName)->is([
+                app($model)->getKeyName(),
+                'created_at',
+                'deleted_at',
+                'updated_at',
+                '*_token',
+            ])) {
+                continue;
+            }
+
+            $columnData = [];
+
+            if (in_array($columnName, [
+                'sku',
+                'uuid',
```

Comment:
`id` column should also be uppercase.

Reply:
We exclude the primary key from generation I think, as its usually readonly

Reply:
Fair.